CHANGELOG.md

```diff
@@ -51,6 +51,7 @@
 - `processor/transform`: Add transformation of logs (#9368)
 - `datadogexporter`: Add `metrics::summaries::mode` to specify export mode for summaries (#8846)
 - `prometheusreceiver`: Add resource attributes for kubernetes resource discovery labels (#9416)
+- `datadogexporter`: Replace HistogramMode defined as string with enum.
```

Comment:
This is technically a breaking change for the Go API, since it will make some code fail to compile: https://go.dev/play/p/KtWLxjhP6az I think it is fine to do this on a single release since it's hard to do over several releases, and the impact is minimal. However, just in case, let's move this to the "🛑 Breaking changes 🛑" section instead of Enhancements.

Reply:
@mx-psi Thanks for your review! Moved to Breaking changes section. https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/9589/commits/c049c656ca153cc5378add68e3351d38b7844e0e